assets/js/googlesitekit/datastore/user/key-metrics.js

```diff
@@ -476,8 +478,14 @@ const baseSelectors = {
 						return false;
 					}
 
-					if ( widget.displayInList ) {
-						return widget.displayInList( select, isViewOnly );
+					if (
+						// The call to displayInList() was added solely for metrics
+						// with custom dimensions, so adding this check till we refactor this
```

Comment:
```suggestion
						// with custom dimensions, so adding this check until we refactor this
```

Reply:
This is going to be removed anyway in #9548 from here as this causes a recursion bug which is being fixed there.